### PR TITLE
Fixes #16983: display openscap report in node details

### PR DESCRIPTION
**Pull request by ncharles**

### CODE REVIEW

**Comment by ncharles** [comment]
https://issues.rudder.io/issues/16983

**Comment by ncharles** [comment]
PR updated with a new commit

File: openscap-policies/src/main/scala/com/normation/plugins/openscappolicies/api/OpenScapApi.scala [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+sealed trait OpenScapApi extends EndpointSchema with GeneralApi with SortIndex
+object OpenScapApi extends ApiModuleProvider[OpenScapApi] {
+
+  final case object GetOpenScapReport extends OpenScapApi with OneParam with StartsAtVersion10 {
+    val z = implicitly[Line].value
+    val description    = "Get OpenScap report for a node"
+    val (action, path) = GET / "openscap" / "report" / "{id}"
+  }
+
+  final case object GetSanitizedOpenScapReport extends OpenScapApi with OneParam with StartsAtVersion15 {
```
Comment (fanf) [inline openscap-policies/src/main/scala/com/normation/plugins/openscappolicies/api/OpenScapApi.scala:38]: it should be version 10

**Comment by ncharles** [comment]
PR updated with a new commit

File: openscap-policies/src/main/scala/com/normation/plugins/openscappolicies/api/OpenScapApi.scala (hunk 1 above)
Comment (ncharles) [reply openscap-policies/src/main/scala/com/normation/plugins/openscappolicies/api/OpenScapApi.scala:38]: actually, it must be 15 :)

**Comment by ncharles** [comment]
PR updated with a new commit

File: openscap-policies/src/main/scala/com/normation/plugins/openscappolicies/api/OpenScapApi.scala (hunk 1 above)
Comment (fanf) [reply openscap-policies/src/main/scala/com/normation/plugins/openscappolicies/api/OpenScapApi.scala:38]: why? It is not in supported API (we stop at 12 in rudder)

Comment (ncharles) [reply openscap-policies/src/main/scala/com/normation/plugins/openscappolicies/api/OpenScapApi.scala:38]: 15 is supported, and all techniques resources and parameters endpoint are already at level 15

**Comment by ncharles** [comment]
PR updated with a new commit

**Comment by fanf** [comment]
OK, squash merging this PR

**Comment by fanf** [comment]
OK, squash merging this PR

**Comment by fanf** [comment]
OK, squash merging this PR

**Comment by fanf** [comment]
I merged it by hand and visibly missed something. IT's merged!

**Comment by fanf** [comment]
It's in commit: https://github.com/Normation/rudder-plugins/commit/aadc961653ee08ea2e8a6e68d1aae06914c14997

**Pull request closed without merging**